modules/migration-understanding-data-copy-methods.adoc

```diff
@@ -3,20 +3,20 @@
 // migration/migrating-3-4/migrating-openshift-3-to-4.adoc
 // migration/migrating-4-4/migrating-openshift-4-to-4.adoc
 [id='migration-understanding-data-copy-methods_{context}']
-= Understanding the file system and snapshot data copy methods
+= Understanding the data copy methods for migration
 
-The CAM tool supports the file system and snapshot data copy methods for migrating data from the source cluster to the target cluster.
+The CAM tool supports the file system and snapshot data copy methods for migrating data from the source cluster to the target cluster. You can select a method that is suited for your environment and is supported by your S3 storage provider.
```

Comment:
Change to:

The CAM tool supports the file system and snapshot data copy methods for migrating data from the source cluster to the target cluster. You can select a method that is suited for your environment and is supported by your storage provider.



--- 
The 'storage provider' determines what is supported, not the Object Store.
i.e. , snapshots have nothing to do with the AWS S3, NooBaa, Azure Blob, Google Cloud Storage, etc.


2 different concepts:
 - Object Store. <-- has no relation to snapshots
 - Volumes <- this is where snapshots occur